internal/cmd/gocdk/internal/static/_assets/demo/demo_secrets.go

```diff
@@ -39,7 +39,6 @@ func init() {
 	// to an local provider that use in-process encryption/decryption.
 	keeperURL = os.Getenv("SECRETS_KEEPER_URL")
 	if keeperURL == "" {
-		// TODO(rvangent): Remove default later.
```

Comment:
Does this default no longer need to be removed?

Reply:
Right -- all of the demos have local implementations as the default.